### PR TITLE
Wiki Cleanup

### PR DESCRIPTION
**Pull request by Pogogo007**
Its wiki cleaning up time! The MR isnt fully done yet im simply posting it so others can comment/suggest/change anything

### CODE REVIEW

File: src/content/docs/general_info/FAQ.mdx [hunk 1]
```diff
@@ -68,24 +68,24 @@ If you install the latest update and find out a specific package doesn't work or
...
 ## System maintanance FAQ
-Here are all the questions related to the system maintenance or changes of configurations of your system.
+Here are all the questions related to the system maintenance or configuration changes of your system.
 
 ### How can I update GRUB after `sudo pacman -Syu`
 This is indicated by the following text in the terminal and usually comes after a `sudo pacman -Syu`: grub packages were updated.
 ```
 :: To use the new features provided in this GRUB update, it is recommended
-   to install it to the MBR or UEFI. Due to potential configuration
+   to install it to the boot partition again. Due to potential configuration
```
Comment (vnepogodin) [inline src/content/docs/general_info/FAQ.mdx]: that is message from grub package. do not change it here

File: src/content/docs/cachyos_repositories/why_cachyos_repo.md [hunk 2]
```diff
@@ -10,20 +10,20 @@ dateCreated: 2021-08-18T15:06:49.466Z
...
-- **Solution:** That error is mostly solved by re-build the package against a newer library and after re-build the package work again.
+- **Solution:** That error is mostly solved by a re-build of the package against a newer library and after the re-build the package works again.
 
 # Tests and benchmarks
-If you can't imagine how much performance you will get, check the links below.  
+If you would like to know more about the performance uplift of our repositories, check the links below.  
 
-- Phoronix tried benchmarks and it seems to be mostly good for CachyOS and showing leading in the performance. These benchmarks are a little bit older, in the time, CachyOS already evolved even more:  
+- Phoronix tried benchmarks and it seems to be mostly good for CachyOS and shows leading in the performance aspect. These benchmarks are a little bit older, during that time, CachyOS has already evolved much more:  
 https://www.phoronix.com/review/cachyos-linux-perf
```
Comment (vnepogodin) [inline src/content/docs/cachyos_repositories/why_cachyos_repo.md:28]: may put also new benchmarks
https://www.phoronix.com/review/cachyos-x86-64-v3-v4

File: src/content/docs/general_info/General_System_Tweaks.mdx [hunk 3]
```diff
@@ -64,10 +64,11 @@ Do so at your own risk.
 
 Add the following to your kernel command line: `retbleed=off` or to disable all mitigations: `mitigations=off`
 
-Edit the appropriate file to persist the changes:
+Edit the appropriate file to make the changes persistent:
 
 *   GRUB: `/etc/default/grub`
 *   systemd boot: `/etc/sdboot-manage.conf`
+*   rEFIndL `/boot/refind_linux.conf`
```
Comment (vnepogodin) [inline src/content/docs/general_info/General_System_Tweaks.mdx]: rEFInd

File: src/content/docs/how_to_install/install-cachyos.md [hunk 4]
```diff
@@ -105,6 +107,6 @@ sudo pacman-key --init; sudo pacman-key --populate; sudo pacman-key --lsign cach
 
 ### Calamares "freezes"
 
-There is since mesa 23.x a issue, that the main GUI thread of calamares has a hardlock on **VMs**.
-The installation itself still gets forward and should be installed fine.
+There is since mesa 23.x an issue, that the main GUI thread of calamares has a hardlock on **VMs**.
+The installation itself still gets forward and should be install fine.
```
Comment (vnepogodin) [inline src/content/docs/how_to_install/install-cachyos.md]: "should install fine."
then

**Review by vnepogodin — CHANGES_REQUESTED** [review]

File: src/content/docs/cachyos_repositories/why_cachyos_repo.md (hunk 2 above)
Comment (Pogogo007) [reply src/content/docs/cachyos_repositories/why_cachyos_repo.md:28]: Ive added them. Do tell me if the format they are in isnt to your liking

Comment (vnepogodin) [reply src/content/docs/cachyos_repositories/why_cachyos_repo.md:28]: seems ok

**Comment by vnepogodin** [comment]
actually it would make sense to replace all string types in `/etc/mkinitcpio.conf` to array, and add replace meaning to add if not added instead of "set"

**Comment by Pogogo007** [comment]
- The option to use Firejail, which runs untrusted applications in a sandbox environment and updates automatically with new packages
- Preconfigured DNS servers that utilize DoH and DoT protocols
- Anonymized DNSCrypt configuration and use of only DNSCrypt servers
- A hardened kernel available in the CachyOS repository
- Security-related packages compiled with extra CC flags for added security

Are all these points still valid?

**Comment by vnepogodin** [comment]
> * The option to use Firejail, which runs untrusted applications in a sandbox environment and updates automatically with new packages

no
> * Preconfigured DNS servers that utilize DoH and DoT protocols

can be applied in Hello
> * Anonymized DNSCrypt configuration and use of only DNSCrypt servers
> 

not anymore

>  * A hardened kernel available in the CachyOS repository
> 

yes
>  * Security-related packages compiled with extra CC flags for added security

yes

**Review by vnepogodin — APPROVED** [review]
LGTM

**Pull request merged**